src/libraries/Microsoft.Extensions.Caching.Memory/README.md

```diff
@@ -0,0 +1,13 @@
+# Microsoft.Extensions.Caching.Memory
+
+In-memory caching provides a general purpose memory implementation of the core caching abstractions provided under `Microsoft.Extensions.Caching.Abstractions` and it is great for apps that run in a single server, where all cached data rents memory in the app's process.
+
+Documentation can be found at https://learn.microsoft.com/en-us/dotnet/core/extensions/caching.
+
+## Contribution Bar
+- [x] [We consider new features, new APIs, bug fixes, and performance changes](../../libraries/README.md#primary-bar)
+
+The APIs and functionality need more investment in the upcoming .NET releases.
+
+## Deployment
+[Microsoft.Extensions.Caching.Memory](https://www.nuget.org/packages/Microsoft.Extensions.Caching.Memory) is not included in the shared framework. The package is deployed as out-of-band (OOB) and needs to be installed into projects directly.
```

Comment:
>is not included in the shared framework. The package is deployed as out-of-band (OOB) and needs to be installed into projects directly. [](http://example.com/codeflow?start=106&length=135)

is included in the ASP.NET Core shared framework. The package is deployed as out-of-band (OOB) too and can be referenced into projects directly.